### PR TITLE
[APB-4895][GW] remove use of static gaClientId

### PR DESCRIPTION
**Pull request by geoffreywatson**

### CODE REVIEW

File: app/uk/gov/hmrc/agentclientauthorisation/service/PlatformAnalyticsService.scala [hunk 1]
```diff
@@ -99,4 +103,14 @@ class PlatformAnalyticsService @Inject()(
       )
     )
 
+  // platform analytics will make a client ID from the session ID but when there is no session make a clientId from a UUID
+  private def makeGAClientId: String = {
+    val uuid = UUID.randomUUID().toString
+    MH3.stringHash(uuid, MH3.stringSeed).abs.toString match {
+      case uuidHash =>
+        "GA.1." + (uuidHash + "000000000")
+          .substring(0, 9) + "." + ("0000000000" + uuidHash).substring(uuidHash.length, 10 + uuidHash.length).reverse
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentclientauthorisation/service/PlatformAnalyticsService.scala:112]: who suggested this implementation or it is a platform recommendation to generate clientId like this ?

File: app/uk/gov/hmrc/agentclientauthorisation/service/PlatformAnalyticsService.scala [hunk 2]
```diff
@@ -42,7 +45,6 @@ class PlatformAnalyticsService @Inject()(
   private val interval = appConfig.invitationUpdateStatusInterval
   private val expiredWithin = interval.seconds.toMillis
   private val batchSize = appConfig.gaBatchSize
-  private val clientId = appConfig.gaClientId
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentclientauthorisation/service/PlatformAnalyticsService.scala:45]: please remove the `gaClientId` from application.conf and app-configs in different environments(if they are configured)

File: app/uk/gov/hmrc/agentclientauthorisation/service/PlatformAnalyticsService.scala (hunk 1 above)
Comment (geoffreywatson) [reply app/uk/gov/hmrc/agentclientauthorisation/service/PlatformAnalyticsService.scala:112]: This is lifted from PA when they generate an ID.

File: app/uk/gov/hmrc/agentclientauthorisation/service/PlatformAnalyticsService.scala (hunk 2 above)
Comment (geoffreywatson) [reply app/uk/gov/hmrc/agentclientauthorisation/service/PlatformAnalyticsService.scala:45]: config removed

**Pull request merged**